src/components/QuestionPlay/WithOptions.tsx

```diff
@@ -0,0 +1,61 @@
+import classNames from 'classnames';
+import React from 'react';
+import { Title, Button, Checkbox } from '@mantine/core';
+import { Option as IOption } from '../../types';
+import Markdown from '../Markdown';
+import styles from './styles.module.css';
+
+interface Props {
+  setSelectedChoice: Function;
+  selectedOptionId: number | null | string;
+  options: IOption[];
+  selectedChoice: number | null | string;
+  isAttempted: boolean;
+  isPlaying: Boolean;
+}
+
+export default function WithOptions({
+  options,
+  setSelectedChoice,
+  selectedOptionId,
+  selectedChoice,
+  isAttempted,
+  isPlaying,
+}: Props) {
+  console.log('selectedOptionId', selectedOptionId);
+  return (
+    <>
+      <Checkbox.Group
+        value={selectedChoice === null ? undefined : [selectedChoice.toString()]}
+        orientation="vertical"
+        label={<Title order={6}>Options</Title>}
+        size="md"
+        my="lg"
+        onChange={(value: string[]) => setSelectedChoice(value[value.length - 1])}>
+        {options.map((option) => (
+          <Checkbox
+            key={option.optionId}
+            label={<Markdown>{option.text}</Markdown>}
+            className={classNames({
+              [styles.isAttempted]: isAttempted,
+              [styles.correct]:
+                option.isCorrect && selectedOptionId && parseInt(selectedOptionId.toString()) === option.optionId,
+              [styles.inCorrect]:
+                !option.isCorrect && selectedOptionId && parseInt(selectedOptionId.toString()) === option.optionId,
+            })}
+            value={option.optionId.toString()}
+            disabled={isAttempted || !isPlaying}
```

Comment:
what is the difference between isAttempted & !isPlaying?

Reply:
isAttempted is mean question is attempted. `isPlaying` mean the timer is running. changing it to isTimerRunning for better understanding